src/python/HTCondorLocator.py

```diff
@@ -122,13 +122,16 @@ def getSchedd(self, chooserFunction=memoryBasedChoices):
         schedd = None
 
         try:
-            htcondor.param['COLLECTOR_HOST'] = collector.encode('ascii', 'ignore')
+            collParam = 'COLLECTOR_HOST'
+            htcondor.param[collParam] = collector.encode('ascii', 'ignore')
             coll = htcondor.Collector()
             # select from collector crabschedds and pull some add values
             # this call returns a list of schedd objects.
             schedds = coll.query(htcondor.AdTypes.Schedd, 'CMSGWMS_Type=?="crabschedd"',
                                  ['Name', 'DetectedMemory', 'TotalFreeMemoryMB', 'TransferQueueNumUploading',
                                   'TransferQueueMaxUploading','TotalRunningJobs', 'JobsRunning', 'MaxJobsRunning', 'IsOK'])
+            if not schedds:
+                raise Exception("No CRAB schedds returned by collecor query. '%s' parameter is '%s'. Try later" %(collParam, htcondor.param['COLLECTOR_HOST']))
```

Comment:
@belforte I wanted to print the collector host used in the query but did not find how to get it. So I printed the entire `'COLLECTOR_HOST'` parameter which is a two-element string currently.

Reply:
I do not think that we have a way to know. WOuld require to call the API twice passing only once collector every time.
Let's leave it at this. We pass the list to the API, the API does its best, but if it returns nohing we want to  avoid printing the wrong message thinking that the list was emptied by one of the follolwing filters instead.